projects/plugins/wpcomsh/README.md

```diff
@@ -11,13 +11,19 @@ With the help of this mu-plugin, an Atomic site is transformed into a WP.com sit
 ### Quick Start
 
 ```
+# Clone the Monorepo somewhere locally
+$ git clone git@github.com:Automattic/jetpack.git
+$ cd jetpack
+$ pnpm install
+$ pnpm jetpack build --deps projects/wpcomsh
+
+# Specify your development server and path to sync the code to your wp.org install 
+$ pnpm jetpack rsync projects/wpcomsh USER@HOST:/path/to/wordpress/wp-content/mu-plugins 
+
 # From the root of a wp.org install
```

Comment:
More specifically, from the root of the wp.org install you just rsynced to.

Alternatively, if we want to assume they're doing this directly on the server like the old instructions did, skip the rsync and have line 26 create a symlink instead.

Reply:
I wouldn't advise people to do it on the server directly, some servers can't handle the monorepo. But you're right, thanks.

Reply:
Maybe something like this?
```suggestion
# From the root of the wp.org install you rsynced to (assuming it didn't already have wpcomsh installed)
```

Reply:
Yeah, that sounds good, thank you!